libmambapy/src/main.cpp

```diff
@@ -54,6 +70,46 @@ deprecated(char const* message)
     warnings.attr("warn")(message, builtins.attr("DeprecationWarning"), py::arg("stacklevel") = 2);
 }
 
+template <typename PyClass>
+auto
+bind_NamedList(PyClass pyclass)
+{
+    using type = typename PyClass::type;
+    pyclass.def(py::init())
+        .def("__len__", [](type const& self) { return self.size(); })
+        .def("__bool__", [](type const& self) { return !self.empty(); })
+        .def(
+            "__iter__",
+            [](type const& self) { return py::make_iterator(self.begin(), self.end()); },
+            py::keep_alive<0, 1>())
+        .def("clear", &type::clear)
+        .def("add", [](type& self, typename type::value_type const& v) { self.insert(v); })
+        .def("name", &type::name)
+        .def("versions_trunc", &type::versions_trunc, py::arg("sep") = "|", py::arg("etc") = "...")
+        .def("build_strings_trunc",
+             &type::build_strings_trunc,
+             py::arg("sep") = "|",
+             py::arg("etc") = "...");
+    return pyclass;
+}
+
+template <typename Node, typename Edge>
+py::object
+make_networkx(mamba::DiGraph<Node, Edge> const& g)
```

Comment:
This requires `networkx`. Perhaps we could return raw data (dicts and list) that can be forwarded (or not) by the user.